cdn/rpcserver/rpcserver.go

```diff
@@ -189,12 +190,13 @@ func (css *Server) GetPieceTasks(ctx context.Context, req *base.PieceTaskRequest
 	for _, piece := range pieces {
 		if piece.PieceNum >= req.StartNum && (count < req.Limit || req.Limit <= 0) {
 			p := &base.PieceInfo{
-				PieceNum:    int32(piece.PieceNum),
-				RangeStart:  piece.PieceRange.StartIndex,
-				RangeSize:   piece.PieceLen,
-				PieceMd5:    piece.PieceMd5,
-				PieceOffset: piece.OriginRange.StartIndex,
-				PieceStyle:  piece.PieceStyle,
+				PieceNum:     int32(piece.PieceNum),
+				RangeStart:   piece.PieceRange.StartIndex,
+				RangeSize:    piece.PieceLen,
+				PieceMd5:     piece.PieceMd5,
+				PieceOffset:  piece.OriginRange.StartIndex,
+				PieceStyle:   piece.PieceStyle,
+				DownloadCost: piece.DownloadCost,
```

Comment:
`PieceInfo` needs to be consistent with the `PieceResult` of  `scheduler.proto`.

https://github.com/dragonflyoss/Dragonfly2/blob/main/pkg/rpc/scheduler/scheduler.proto#L96
https://github.com/dragonflyoss/Dragonfly2/blob/main/pkg/rpc/scheduler/scheduler.proto#L98